src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml

```diff
@@ -42,8 +42,8 @@ tests:
 
     - label: "TH reads the FeatureMap from DUT"
       PICS:
-          " !DGTHREAD.S.F00 && !DGTHREAD.S.F00 && !DGTHREAD.S.F00 &&
-          !DGTHREAD.S.F00 "
+          " !DGTHREAD.S.F00 && !DGTHREAD.S.F01 && !DGTHREAD.S.F02 &&
```

Comment:
DGTHREAD.S.F00 and DGTHREAD.S.F01 do not exist in PICS v0.11 xml.
are we updating PICS xml or yaml?

Reply:
@yinyihu-silabs these changes are as part of recent updates in the test plan  [PR](https://github.com/CHIP-Specifications/chip-test-plans/pull/2216) @raju-apple

Reply:
@yinyihu-silabs 
![image](https://user-images.githubusercontent.com/24218865/189411816-b4b62f00-1d18-46f3-b866-832f62fed130.png)